### PR TITLE
[Do Not Merge] Bug 1225046 - Marionette test for launching FxA from Sync, r=ferjm

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5a443a18588ee49e23ea45e22f50479cafe8b570

File: apps/system/fxa/test/marionette/lib/fxa.js [hunk 1]
```diff
@@ -149,9 +151,19 @@ FxA.prototype = {
...
       var element = this.client.findElement(FxA.Selectors.fxaLogin);
+      console.log('found element', element);
       this.client.helper
           .waitForElement(element)
           .tap();
+      console.log('tapped');
+    },
+    clickFxsyncLogin: function() {
+      var element = this.client.findElement(FxA.Selectors.fxsyncLogin);
+      console.log('found element', element);
```
Comment (michielbdejong) [inline apps/system/fxa/test/marionette/lib/fxa.js]: This works, it finds the element.

File: apps/system/fxa/test/marionette/lib/fxa.js [hunk 2]
```diff
@@ -149,9 +151,19 @@ FxA.prototype = {
...
+      this.client.helper
+          .waitForElement(element)
+          .tap();
+      console.log('tapped');
```
Comment (michielbdejong) [inline apps/system/fxa/test/marionette/lib/fxa.js]: This does not have the effect of opening the FxA iFrame. When I do `document.querySelector('#get-started').click();` from the console, it does open the iFrame, so I'm not sure what's happening here...

Comment (rpappalax) [inline apps/system/fxa/test/marionette/lib/fxa.js]: not sure what's going on, but did you try book-ending the get-started click with a few sleeps to see if there's a timing issue not being accounted for by waitForElement?  marionette often leaves me scratching my head.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=05e89d03252329c5b1d2364b7918deeaa692459f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=18be6b4098dbb5783d7b768a174cbcc7d740daec

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dc4bed2bffa6b2109659927713de6adb874ee6a7

**Comment by michielbdejong** [comment]
Cleaning up PRs that are > 1 year old.

**Pull request closed without merging**